### PR TITLE
Lessons 1 and 2, modified files

### PR DESCRIPTION
**Pull request by artpetrovskii**

### CODE REVIEW

File: lesson 1. exercise 3.py [hunk 1]
```diff
@@ -10,8 +10,6 @@
             break
         else:
             print("Введено некорректно значение. Попробуйте еще раз")
-    except:
```
Comment (apetrovskiy) [inline lesson 1. exercise 3.py]: It doesn't work

File: lesson 2. exercise 1.py [hunk 2]
```diff
@@ -1,7 +1,4 @@
 list1 = ['4', 'dfsdf', 'kh43', 'vlof', 'dfsdf']
 list2 = ['456', 'dfsdf', '89']
 
-for item in list2:
-    if item in list1:
-        list1.remove(item)
-print(list1)
+print([k for k in list1 if list2.count(k) < 1])
```
Comment (apetrovskiy) [inline lesson 2. exercise 1.py:4]: image

File: lesson 2. exercise 2.py [hunk 3]
```diff
@@ -15,13 +15,17 @@
...
+    '05': 'пятое', '06': 'шестое', '07': 'седьмое', '08': 'восьмое',
+    '09': 'девятое', '10': 'десятое', '11': 'одиннадцатое',
+    '12': 'двенадцатое', '13': 'тринадцатое', '14': 'четырнадцатое',
+    '15': 'пятнадцатое', '16': 'шестнадцатое', '17': 'семнадцатое',
+    '18': 'восемнадцатое', '19': 'девятнадцатое', '20': 'двадцатое',
+    '21': 'двадцать первое', '22': 'двадцать второе', '23': 'двадцать третье',
+    '24': 'двадцать четвёртое', '25': 'двадцать пятое',
+    '26': 'двадцать шестое', '27': 'двадцать седьмое',
+    '29': 'двадцать девятое', '28': 'двадцать восьмое',
+    '30': 'тридцатое', '31': 'тридцать первое',
```
Comment (apetrovskiy) [inline lesson 2. exercise 2.py]: dana_data = '03.11.2019'
and
dana_data = '3.11.2019'
return different resutls

**Review by apetrovskiy — CHANGES_REQUESTED** [review]

File: lesson 2. exercise 2.py (hunk 3 above)
Comment (artpetrovskii) [reply lesson 2. exercise 2.py]: I resolved this mistake

**Review by apetrovskiy — APPROVED** [review]

**Review by apetrovskiy — APPROVED** [review]

**Pull request merged**